crates/analytics/src/auth_events/metrics/challenge_success_count.rs

```diff
@@ -56,36 +49,23 @@ where
                 .switch()?;
         }
 
-        filters.set_filter_clause(&mut query_builder).switch()?;
-
-        query_builder
-            .add_filter_clause("merchant_id", publishable_key)
-            .switch()?;
-
         query_builder
-            .add_filter_clause("event_name", SdkEventNames::ThreeDsMethodResult)
+            .add_filter_clause("merchant_id", merchant_id)
             .switch()?;
 
         query_builder
-            .add_filter_clause("log_type", "ERROR")
+            .add_filter_clause("flow", AuthEventFlows::PostAuthentication)
             .switch()?;
 
         query_builder
-            .add_filter_clause("category", "USER_EVENT")
+            .add_filter_clause("visitParamExtractRaw(response, 'transStatus')", "\"Y\"")
```

Comment:
why not extract this out as a separate field in the mv?

Reply:
The usecase is too specific to authentication analytics.
Since the other filters anyway restrict the data scanned, this should be fined